### PR TITLE
Add syncing message and disable sends while chain is syncing

### PR DESCRIPTION
**Pull request by dgca**
- Adds syncing message to accounts list and send form if chain is syncing.
- Adds tooltip to send button in account row.

Fixes IFL-1896

### CODE REVIEW

**Comment by dgca** [comment]
@dguenther Heyo, curious if you have any suggestions here. This PR adds makes it so that the message appears/disappears briefly here and there while the chain goes out of sync and re-syncs. It's not an ideal experience but it will like randomly disable the Send button for a split second every once in a while.

Can you think of a better way to implement this? Are we able to check if the chain is behind by X number of blocks, or check if an individual account is out of sync?

**Comment by dguenther** [comment]
I'll take a look at what the CLI does for this.

**Comment by dguenther** [comment]
Looks like in order for the CLI to allow a send, it checks whether:

[`getNodeStatus().blockchain.synced` is true](https://github.com/iron-fish/ironfish/blob/c8343c29d143d6855b2b9c6da6c7fc526e9416b2/ironfish-cli/src/commands/wallet/send.ts#L112C20-L112C20)

and

whether the wallet's head hash (`getNodeStatus().accounts.head.hash`) equals the account's head hash (`getAccountStatus().account.head.hash`): https://github.com/iron-fish/ironfish/blob/c8343c29d143d6855b2b9c6da6c7fc526e9416b2/ironfish/src/wallet/wallet.ts#L1728

I think for now we should go with this too so the functionality is similar in both places. This should cut down on the message popping up, since usually the account's head hash will be in sync with the wallet's head hash.

File: renderer/utils/useSyncStatus.ts [hunk 1]
```diff
@@ -0,0 +1,49 @@
+import { useMemo } from "react";
+
+import { trpcReact } from "@/providers/TRPCProvider";
+
+type Status = "CONNECTING" | "SYNCING" | "SCANNING" | "SYNCED";
+
+export function useSyncStatus() {
```
Comment (dguenther) [inline renderer/utils/useSyncStatus.ts:7]: I don't think this is used anywhere, can this be removed?

**Review by dguenther — APPROVED** [review]

**Pull request merged**